README.md

```diff
@@ -1 +1,87 @@
-# Catalogue-Ruby-Capstone
\ No newline at end of file
+## Catalogue Ruby Capstone
+
+This app is develped in ruby and allows users to:
+
+- Ruby console app with an user interface.
+- You can add books or music albums, and you can create a label or genre for the said item.
+- The application preserves user data by using JSON files while exiting the application. 
+
+## Built With
+
+- Ruby
+- OOP
+- Catalogue
+
+### Technologies used
+
+- Github
+- GitFlow
+- Visual Studio Code
+- rvm
+- Rspec
+
+## `Getting Started`
+
+To get a local copy of this project:
+
+Clone this repository or download the Zip folder:
+```
+git clone https://github.com/ShoiraTa/ruby-capstone.git
+```
+
+To get started, In the root directory run:
+```
+gem install bundler
+```
+Then run:
+```
+bundle install
+```
+run ruby app in the Command Line:
+```
+ruby main.rb
+```
+
+### Run tests
+
+- Open the terminal and run `rspec spec spec/filename.rb`
+
+
+### To track linter errors locally follow these steps:  
+
+Track Ruby linter errors run:
+```
+rubocop
+```
+To auto-correct Rubocop offenses run:
+```
+rubocop --auto-correct-all | rubocop -A
+```
+
+## Authors
+
+👤 **Sumair Qaisar**
+
+- GitHub: [@sumairq](https://github.com/sumairq)
+- Twitter: [@sumair_qaisar](https://twitter.com/sumair_qaisar)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/sumairq/)
+
+
+
+👤 **Muhammad Rizo Abdunazarov**
+
+- GitHub: [@mukhammadrizooff](https://github.com/mukhammadrizooff)
+- Twitter: [@Muhammadrizooff](https://twitter.com/Muhammadrizooff)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/mukhammadrizooff/)
+
+## Acknowledgments
+
+- Microverse
+
+## 📝 License
+
+This project is [MIT](LICENSE) licensed.
+
+## Show your support
+
+Give a ⭐️ if you like this project!
```

Comment:
- [ ] Hello. Hope you guys are fine. Please you have forgotten to add a link to your video presentation. Please try and add it.

Reply:
Thank you for your feedback. We added the video link to the presentation just now.